lesson02/assignment/css/style.scss

```diff
@@ -4,3 +4,97 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+.main{
+  background-color: $gray;
+  width: 951px;
+  height: 474px;
+
+//media query mixin to change first background height and width for screen//
+  @include $mq {
+    width: auto;
+    height: 100%;
+  }
+
+  &__first-background{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding: 1em;
+    background-color: $white;
+    width: 847px;
+    height: 383px;
+
+  //media query mixin to change flex direction to columns and height and width for screen//
+    @include $mq{
+      display: flex;
+      flex-direction: column;
+      flex-wrap: wrap;
+      width: auto;
+      height: 100%;
+    }
+  }
+  &__head-title{
+    padding: 1em 0 1em 0;
+    text-align: center;
+    width: 40%;
+    align-self: center;
+
+  //media query mixin to change padding for screen//
+    @include $mq{
+      padding: 1em;
+      width: auto;
+    }
+  }
+  &__h1{
+    font-size: 26px;
+    color: darken($gray, 40%);
+  }
+  &__p{
+    color: darken($gray, 20%);
+    font-style: italic;
+    font-size: 20px;
```

Comment:
Setting font size to a static px size will make life really difficult for low-vision users who need to zoom in to read text. When setting font size, always use % or rem (which are effectively the same thing, as "rem" is root font size).

Reply:
Thank you!!